debezium-connector-oracle/pom.xml

```diff
@@ -70,6 +70,22 @@
             <artifactId>xmlparserv2</artifactId>
             <scope>provided</scope>
         </dependency>
+        <!-- Dependencies for OpenLogReplicator -->
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-core</artifactId>
+            <version>4.2.4</version>
```

Comment:
Please eternalize the version into BOM or if not possible at least use system property.

Reply:
Thanks for pointing this out.  I ended up not using vert.x and there was only one use of it with `Buffer` while sending data to OpenLogReplicator.  I've replaced that code and removed vert.x as a dependency.